gcp_variant_transforms/pipeline_common.py

```diff
@@ -151,7 +149,7 @@ def _get_file_names(input_file):
   if not filesystems.FileSystems.exists(input_file):
     raise ValueError('Input file {} doesn\'t exist'.format(input_file))
   with filesystems.FileSystems.open(input_file) as f:
-    contents = map(str.strip, f.readlines())
+    contents = list(map(str.strip, [b.decode('utf-8') for b in f.readlines()]))
```

Comment:
Here we used to set `contents` to be a map and now it's a list, is this correct?

Also let's replace `b` with `l`.

Reply:
I did it for pylint, so yeah, it's required.
Done.